helpers.py

```diff
@@ -1,8 +1,11 @@
 '''Module containing various helper functions'''
 
+# File specific rules
+# pylint: disable=W0718
+
 # system imports
 import functools
-import re
+import re   # pylint: disable=unused-import
```

Comment:
why not remove it if unused?

Reply:
I wasn't sure if this leftover was still there for some reason 🤔

Reply:
I don't know -> remove it